.travis.yml

```diff
@@ -21,6 +21,7 @@ php:
   - 5.4
   - 5.6
   - 7.0
+  - nightly
```

Comment:
yes doc looks bit old, but if you check https://travis-ci.org/doctrine/DoctrineORMModule and https://github.com/doctrine/DoctrineORMModule/blob/master/.travis.yml#L48
there you can see that php 7.2 already added,